### PR TITLE
bug fixes/adding features

### PR DESCRIPTION
**Pull request by hassanelsherbiny**
adding search in admin panel menu
fixing some bugs in rtl

### CODE REVIEW

File: Grand.Web/Areas/Admin/Views/Shared/Menu.cshtml [hunk 1]
```diff
@@ -18,6 +18,29 @@
         <div class="main_menu_side hidden-print main_menu" id="sidebar-menu">
             <div class="menu_section">
                 <ul class="nav side-menu page-sidebar-menu">
+                    <li class="sidebar-search-wrapper" id="SearchLi">
```
Comment (PatrykPorabik) [inline Grand.Web/Areas/Admin/Views/Shared/Menu.cshtml:21]: May I ask you what's the purpose of adding it to the GrandNode?

Comment (hassanelsherbiny) [reply Grand.Web/Areas/Admin/Views/Shared/Menu.cshtml:21]: to add search textbox to admin panel menu allowing the user to fast filter menu to search what ever he want

Comment (PatrykPorabik) [reply Grand.Web/Areas/Admin/Views/Shared/Menu.cshtml:21]: But it's possible with the admin search on top.

Comment (hassanelsherbiny) [reply Grand.Web/Areas/Admin/Views/Shared/Menu.cshtml:21]: you are right, i didn't notice this search . i think i had the idea from previous version but added it now so i couldn't see that you have implemented a better solution. you can rollback this change

Comment (PatrykPorabik) [reply Grand.Web/Areas/Admin/Views/Shared/Menu.cshtml:21]: Please remove that changes from pull request and send it again.

**Pull request closed without merging**